test/src/main/java/io/strimzi/test/TestUtils.java

```diff
@@ -433,5 +437,40 @@ public static String getContent(File file, Function<JsonNode, String> edit) {
             return Collections.emptyMap();
         }
     }
+
+    /**
+     * Repeat request n-times in a row
+     *
+     * @param retry count of remaining retries
+     * @param fn    request function
+     * @return
+     */
+    public static <T> T doRequestNTimes(int retry, Callable<T> fn, Optional<Runnable> reconnect) throws Exception {
```

Comment:
Why do we need to send request 10 times in MsgCliApiClient? Sending one request is not enough?

Reply:
The html request sometimes fails, this should workaround this. It send one request and based on the result it will sent another or end.